docs/docs/administer/upgrade.md

```diff
@@ -21,8 +21,18 @@ issue](https://github.com/firezone/firezone/issues/new/choose).
 
 ## Upgrading from < 0.5.0 to >= 0.5.0
 
-Firezone has removed support for pre-configured Okta and Google OAuth2 providers.
-Follow the instructions below based on your current setup to migrate to OIDC providers:
+In order to support user-scoped rules Firezone now requires that no two firewall
+rules within the deny or allow lists fall within the same network range.
+So when upgrading firezone will keep the most general rule.
+
+e.g. If you had in the deny-list the following rules:
+10.0.1.0/24, 10.0.0.0/16 and 10.0.1.97 we will keep 10.0.0.0/16.
+
+Furthemore, Firezone has removed support for pre-configured Okta and Google
```

Comment:
I'd put this under a subheader as well, e.g.

### Preconfigured Okta and Google SSO

Firezone 0.5.0 removes support for the old-style Okta and Google SSO configuration in favor of the new, more flexible OIDC-based configuration. If you have any configuration under the `default['firezone']['authentication']['okta']` or `default['firezone']['authentication']['google']` keys, **you need to migration these to our OIDC-based configuration using the guide below.**